CMakeLists.txt

```diff
@@ -1,17 +1,27 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(ign-rviz)
+add_compile_options(-std=c++14 -O2)
```

Comment:
why are you adding the flag -02 ?

Reply:
Initially, when I was writing the code for PointCloud2 visualization, it was too slow. I added compiler optimization to check if there is any significant boost in the code. I'll remove this if not required.